frontend/packages/console-app/src/components/guided-tours/TaskHeader.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
+import cx from 'classnames';
+import { Title } from '@patternfly/react-core';
+import { TaskState } from './utils/guided-tour-status';
+import './TaskHeader.scss';
+
+type TaskHeaderProps = {
+  title: string;
+  taskIndex: number;
+  subtitle?: string;
+  taskState?: TaskState;
+  size?: 'md' | 'lg' | 'xl' | '2xl' | '3xl' | '4xl';
+};
+
+const TaskHeader: React.FC<TaskHeaderProps> = ({ title, taskIndex, subtitle, taskState, size }) => {
+  const TaskIcon: React.FC = () => {
+    switch (taskState) {
+      case TaskState.SUCCESS:
+        return (
+          <span className="co-icon-and-text__icon">
+            <i className="fas fa-check-circle" />
+          </span>
+        );
+      case TaskState.FAILED:
+        return (
+          <span className="co-icon-and-text__icon">
+            <i className="fas fa-times-circle" />
+          </span>
+        );
+      default:
+        return (
+          <span className="co-icon-and-text__icon co-quick-start-task-header__task-icon">
+            {taskIndex}
+          </span>
+        );
+    }
+  };
+  return (
+    <div style={{ marginBottom: 'var(--pf-global--spacer--md)' }}>
+      <Title
+        headingLevel="h5"
+        size={size || 'lg'}
+        className={cx('co-quick-start-task-header__title', {
+          'co-quick-start-task-header__title-success': taskState === TaskState.SUCCESS,
+          'co-quick-start-task-header__title-failed': taskState === TaskState.FAILED,
+        })}
+      >
+        <TaskIcon />
+        {title}
+      </Title>
```

Comment:
Same as mentioned above the related styles. You are using the Title at multiple places, best way would be to create a component that takes State FAILED/SUCCESS as a prop and based on that render the title with your styles.

Reply:
couldn't use Title here as it throws an error due h4 being a child of h4